src/conditionals/nested_statements/q12/MainTest.java

```diff
@@ -0,0 +1,33 @@
+package conditionals.nested_statements.q12;
```

Comment:
there should also be a package between nested_statements and q12 for difficult. I didn't spot this in the other pr but it applies there as well